### PR TITLE
Banner update

### PR DESCRIPTION
**Pull request by insacuri**
# Motivation and Context

# What has changed

# How to test?

# Links

# Screenshots (if appropriate):

### CODE REVIEW

File: frontstage/controllers/banner_controller.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    :return: The banner text, if any, else an empty string
+    """
+    url = f"{app.config['BANNER_SERVICE_URL']}/banner"
+    response = requests.get(url)
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        if response.status_code != 404:
+            logger.error('Failed to retrieve Banner from api')
+        return ""
```
Comment (warrenbailey) [inline frontstage/controllers/banner_controller.py:24]: might be worth logging the status code with the error

**Review by warrenbailey — APPROVED** [review]

File: frontstage/controllers/banner_controller.py (hunk 1 above)
Comment (insacuri) [reply frontstage/controllers/banner_controller.py:24]: Yeah good idea

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**